spec/features/cards_managment_spec.rb

```diff
@@ -14,11 +14,11 @@
     expect(page).to have_text("Новая карточка создана")
   end
   
-  # it "checks translation" do
-  #   visit root_path
-  #   
-  #   fill_in :translate, with: "Home"
-  #   click_button 'Проверить!'
-  #   expect(page).to eq root_path
-  # end
+  it "checks translation" do
+    visit root_path
+    
+    fill_in :translate, with: "Home"
+    click_button 'Проверить!'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.